static/app/components/events/eventStatisticalDetector/eventDifferentialFlamegraph.tsx

```diff
@@ -567,6 +569,77 @@ const DifferentialFlamegraphChangedFunctionContainer = styled('div')`
   }
 `;
 
+interface DifferentialFlamegraphExplanationBarProps {
+  negated: boolean;
+}
+function DifferentialFlamegraphExplanationBar(
+  props: DifferentialFlamegraphExplanationBarProps
+) {
+  return (
+    <DifferentialFlamegraphExplanationBarContainer>
+      <div>
+        {props.negated
+          ? `Flamegraph is showing how stack frequency will change.`
+          : `Flamegraph is showing how stack frequency has changed.`}
```

Comment:
Let's make sure to wrap this in `t(...)`

Reply:
💯